pyproject.toml

```diff
@@ -31,13 +31,11 @@ exclude = [
 ]
 
 # Same as Black.
-line-length = 88
+line-length = 120
```

Comment:
I guess we prefer 88 here

Reply:
I think before we started the migration to ruff we had 120 (see original PR). 
I can of course change it to 88. I might need to format some code a bit if I remember correctly. Is that on ?

Reply:
If I set `88` I get `Found 821 errors` (this comes from the 120 rule I mentioned). I could either keep `120` or set it to `88` and ignore the `E501` rule as it is very hard to fix 821 errors :)

Reply:
Previously, we allowed doc length 120 and code length 88. Do you think that we can maintain that behavior? If we set 120 here, will long code be formatted?

Reply:
If we set 120 there is not refactor needed at all. I guess we can simply add 88 and ignore the rule for docstrings

Reply:
Seems like
```
[tool.ruff.lint.pycodestyle]
max-line-length = 120
```
does the job

Reply:
Indeed! Thanks 💪 ! See https://github.com/pyro-ppl/numpyro/pull/1720/commits/493da8eff5fdee5ee33c6d4f64883e456b1dcff6